jcl/src/java.base/share/classes/openj9/tools/attach/diagnostics/base/DiagnosticProperties.java

```diff
@@ -274,4 +321,43 @@ public static String printProperties(final Properties props) {
 	public Properties toProperties() {
 		return baseProperties;
 	}
+
+	/**
+	 * @return Properties file indicating success
+	 */
+	public static DiagnosticProperties makeCommandSucceeded() {
+		return makeStatusProperties(false, ""); //$NON-NLS-1$
```

Comment:
Does the message need to be an empty String? It appears null is supported.

Reply:
True.